src/FakeItEasy/ArgumentConstraintManagerExtensions.cs

```diff
@@ -222,7 +221,7 @@ public static T Matches<T>(this IArgumentConstraintManager<T> manager, Func<T, b
         {
             Guard.AgainstNull(manager, nameof(manager));
 
-            return manager.Matches(predicate, x => x.Write(string.Format(CultureInfo.CurrentCulture, descriptionFormat, args)));
+            return manager.Matches(predicate, x => x.Write(string.Format(descriptionFormat, args)));

```

Comment:
`string.Format` is redundant, it can be written as `x.Write(descriptionFormat, args)`